geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -106,6 +106,7 @@
   @RequestMapping(method = RequestMethod.GET,
       value = REGION_CONFIG_ENDPOINT + "/{regionName}/indexes")
   @ResponseBody
+  @PreAuthorize("@securityService.authorize('CLUSTER', 'READ', 'QUERY')")
```

Comment:
Should this product change be part of a test reorg?

Reply:
The test reorg also reveals some controller did not enforce permissions.